great-idea/index.html

```diff
@@ -14,10 +15,81 @@
   <![endif]-->
 </head>
 
-  <body>
+<body>
+
+  <nav>
+    <a href="#">Features</a>
+    <a href="#">About</a>
+    <a href="services.html"></a>">Services</a> <!--target="_blank"-->
```

Comment:
The services here is broken. Remember to look over the finished page in the web preview when you are done! But great job making it a link.